app/src/androidTest/java/com/sdp/movemeet/Activity/ActivityNotNullDescriptionTest.java

```diff
@@ -0,0 +1,65 @@
+package com.sdp.movemeet.Activity;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.test.espresso.intent.Intents;
+import androidx.test.ext.junit.rules.ActivityScenarioRule;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.rule.ActivityTestRule;
+
+import com.sdp.movemeet.R;
+import com.sdp.movemeet.Sport;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+;import java.util.ArrayList;
+import java.util.Date;
+
+import static androidx.test.core.app.ApplicationProvider.getApplicationContext;
+
+@RunWith(AndroidJUnit4.class)
+public class ActivityNotNullDescriptionTest {
```

Comment:
What's this test class for?

Reply:
This is to test an activity display when the activity is instantiated (via nu intent). For the moment, the test just launches the display of the description with a non-null activity. It would be interesting to improve it to check the coherence of the transmitted information